index.html

```diff
@@ -192,6 +192,38 @@ <h3>Skills</h3>
         </div>
         
     </section>
+
+    <!-- Contact Section -->
+    <section class="contact-section">
+        <h2>I'm always interested in hearing about new projects, 
+            so if you'd like to chat please get in touch.</h2>
+            
+        <form action="https://formspree.io/f/xknagbvo" method="post" id="contact-form">
+            <input type="text" name="name" id="client-name" maxlength="30" placeholder="Full Name" required>
+            <input type="email" name="email" id="client-email" placeholder="Email Address" required>
+            <textarea name="msg" id="client-msg" cols="30" rows="5" maxlength="500" placeholder="Leave a message" required></textarea>
+            <button class="action-btn" type="submit">Get in touch</button>
+        </form>
```

Comment:
- [ ] Great project so far there are just a few minor changes we need to sort out and you'll be well on your way :wink: 
- [ ] Your button looks great :smile: However, it is missing one of the button interaction styles. I have attached screenshots below. Kindly fix these details so you can completely meet all the design guidelines :100: 
- Your Design when pressed
![image](https://user-images.githubusercontent.com/102699958/213013887-6ce09d60-eba1-4b42-b5ce-6de935e704d3.png)

- The Figma Design when pressed
![image](https://user-images.githubusercontent.com/102699958/213013988-6828102f-e12b-4731-9e70-93311d760048.png)

Reply:
Hi @NahnahAJ ,

Thank you for the feedback!
I appreciate your review and the suggested changes 🙏 

I'm currently working on them to improve the code 😎